spec/features/admin/inviting_a_user_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+feature 'as system admin, inviting a user' do
+  scenario 'switching between accounts with and without free quota', js: true do
+    pageflow_configure do |config|
+      config.quotas.register(:users, QuotaDouble.account_exhaustible)
+    end
+
+    account_with_quota = create(:account)
+    account_without_quota = create(:account, name: 'Exhausted account')
```

Comment:
The above quota double is a bit too magic for my taste. By only looking at the test, I never would have guessed that it's the name of the account that causes the quota to be exhausted for the one account and available for the other.

Maybe `QuotaDouble.account_exhaustible` can receive a mapping from account to desired quota state etc. Alternatively I would at least give the method a really descriptive name like `QuotaDouble.determine_state_by_account_name` to point out what is going on here. Still, specifying a mapping feels simpler and more expressive.